Formula/h/homeworlds.rb

```diff
@@ -25,6 +25,9 @@ class Homeworlds < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "88f7a3976037aea175f742cb16c1e3e0e7e0f7945bf9f24b42eaedeabe16834c"
   end
 
+  # Unversioned and upstream repository has no tags or releases.
+  deprecate! date: "2025-01-26", because: "does not meet requirements for `stable` URLs"
```

Comment:
Can we ask upstream if they're willing to cut a release tag before deprecating this?

Reply:
Sure, let's see if we get a response: https://github.com/Quuxplusone/Homeworlds/issues/5

That said, it's worth acknowledging that we're asking upstream to version an unversioned project and that may not be something that they want to do (especially considering it hasn't been updated in a few years). There's no harm in asking but we should temper our expectations for these projects.